dipy/segment/quickbundles.py

```diff
@@ -5,102 +5,110 @@
 from warnings import warn
 
 warn(DeprecationWarning("Module 'dipy.segment.quickbundles' is deprecated,"
-			" use module 'dipy.segment.clustering' instead"))
+                        " use module 'dipy.segment.clustering' instead"))
 
 
 class QuickBundles(object):
-    
-    def __init__(self,tracks,dist_thr=4.,pts=12):
-        """ Highly efficient trajectory clustering 
-        
+
+    def __init__(self, tracks, dist_thr=4., pts=12):
+        """ Highly efficient trajectory clustering
+
         Parameters
         -----------
         tracks : sequence of (N,3) ... (M,3) arrays,
                     trajectories (or tractography or streamlines)
-                    
-        dist_thr : float, 
+
+        dist_thr : float,
                     distance threshold in the space of the tracks
-        pts : int, 
-                number of points for simplifying the tracks 
-                       
+        pts : int,
+                number of points for simplifying the tracks
+
         Methods
         --------
         clustering() returns a dict holding with the clustering result
         virtuals() gives the virtuals (track centroids) of the clusters
-        exemplars() gives the exemplars (track medoids) of the clusters        
-        
+        exemplars() gives the exemplars (track medoids) of the clusters
+
         Citation
         ---------
-        
-        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012 
-        
+
+        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
```

Comment:
@arokem I talked with @Garyfallidis, he said to leave the citation to his Ph.D. thesis as we will remove this file at some point anyway (after the `dipy_quickbundle` script is modified).